dist/loginApp/loginApp.js

```diff
@@ -90,8 +90,8 @@
                         if (search.state) {
                             scope.changeCurrentForm(search.state);
                         }
-                        $location.search('app', null);
-                        $location.search('state', null);
+                        // $location.search('app', null);
```

Comment:
in the meantime no, but it was a quickfix, in the future i guess you would want to remove that url parameters, so i left the comment there